decbinhex4/config.nims

```diff
@@ -0,0 +1,26 @@
+## This is applied to all workspaces below it, thus applies to both extensions.
+import os
+
+--app: lib
+--noMain: on
+--define: nimPreviewDotLikeOps
+
+let workspaceName = projectDir().splitFile.name
+
+switch("define", "projectName:" & workspaceName)
```

Comment:
projectName: has been changed to ExtensionName: in the latest version.

```nim
switch("define", "ExtensionName:" & workspaceName)
```

Reply:
I still need to `let` workspaceName though, I assume?